lib/Backend/Lower.cpp

```diff
@@ -10883,6 +10900,22 @@ Lowerer::LowerBailOnNotObject(IR::Instr       *instr,
 }
 
 IR::Instr *
+Lowerer::LowerBailOnTrue(IR::Instr      *instr,
+                                 IR::LabelInstr *labelBailOut /*nullptr*/)
```

Comment:
Formatting here looks pretty strange. You could probably put the params on one line since the function name is shorter.